custom/enikshay/case_utils.py

```diff
@@ -294,6 +294,19 @@ def get_adherence_cases_between_dates(domain, person_case_id, start_date, end_da
     return open_pertinent_adherence_cases
 
 
+def get_associated_episode_case_for_test(domain, test_case):
```

Comment:
To follow the convention of the other names in this file, do you think this should be `get_episode_case_from_test`?

Reply:
There is a little difference in what this method does. It does not do any traversal but gets the episode case id from the test case itself so named it differently.